tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -370,7 +370,9 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
 
 ```bash
-$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow
+$ docker run -d -p 8888:8888 --name="tensor" gcr.io/tensorflow/tensorflow
+
+$ docker exec -it tensor bash
```

Comment:
Sure, that sounds fine -- as long as these cases are documented well (why you would use one vs. the other), having both instructions seems helpful.  We don't want to embed all of docker documentation in TF, so we should link to Docker documentation where it makes sense